static/styles/zulip.css

```diff
@@ -3053,3 +3053,40 @@ select.inline_select_topic_edit {
         margin: 0;
     }
 }
+
+#scroll-to-bottom-button-container {
+    display: none;
+    z-index: 3;
+    position: absolute;
+    bottom: 41px;
+    right: 0;
+
+    #scroll-to-bottom-button-clickable-area {
+        width: 60px;
+        height: 60px;
```

Comment:
@terpimost  I added a 60x60 clickable area. The actual button radius is 40px though. 140px is too big. I don't know if it is good have this invisible clickable area though.